CMakePlugins.cmake

```diff
@@ -12,16 +12,6 @@
 option (PLUGIN_USE_EXTERNAL "Use External Library Building for filter PLUGIN else search" OFF)
 option (PLUGIN_USE_LOCALCONTENT "Use local file for PLUGIN FetchContent" OFF)
 
-if (NOT PLUGIN_USE_LOCALCONTENT)
-  set (PLUGIN_URL ${PLUGIN_TGZ_ORIGPATH}/${PLUGIN_TGZ_NAME})
-else ()
-  if (NOT H5PL_TGZPATH)
-    set (H5PL_TGZPATH ${TGZPATH})
-  endif ()
-  set (PLUGIN_URL ${H5PL_TGZPATH}/${PLUGIN_TGZ_NAME})
-endif ()
-message (VERBOSE "Filter PLUGIN file is ${PLUGIN_URL}")
-
```

Comment:
Should commented line 16 #option... be removed?

Reply:
No that is an informational comment for the global - defined elsewhere option